integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPodsRestart.java

```diff
@@ -401,11 +495,33 @@ public void testServerPodsRestartByChaningPodSecurityContext() {
     assertTrue(verifyRollingRestartOccurred(podsWithTimeStamps, 1, domainNamespace),
         String.format("Rolling restart failed for domain %s in namespace %s", domainUid, domainNamespace));
 
+    logger.info("verify domain roll starting/pod cycle starting events are logged");
+    //************* I don't see this event logged****************************************
```

Comment:
Put the pending  issue to test description.

Reply:
added.